rabbitmq/tests/test_openmetrics.py

```diff
@@ -393,3 +394,132 @@ def test_aggregated_endpoint(aggregator, dd_run_check, mock_http_response):
         metric_type=aggregator.MONOTONIC_COUNT,
         at_least=23,
     )
+
+
+def test_bare_detailed_endpoint(aggregator, dd_run_check, mock_http_response):
+    """User enables only the /metrics/detailed endpoint and without any query to it.
+
+    We expect at least metrics like build_info and identity_info to still be present.
+    """
+    mock_http_response(file_path=OPENMETRICS_RESPONSE_FIXTURES / "detailed.txt")
+    check = RabbitMQ(
+        "rabbitmq",
+        {},
+        [
+            {
+                'prometheus_plugin': {
+                    'url': "localhost:15692",
+                    'unaggregated_endpoint': 'detailed',
+                },
+                "metrics": [".+"],
+            }
+        ],
+    )
+    dd_run_check(check)
+    expected_metrics = [
```

Comment:
Same comment as one in an earlier PR, to use the built-in aggregator assert methods to simplify the test: https://github.com/DataDog/integrations-core/pull/13684#discussion_r1069772419

Reply:
See #13737